vignettes/prudence.Rmd

```diff
@@ -60,14 +57,14 @@ df$y
 nrow(df)
 ```
 
-Under the hood, two key differences provide improved performance and usability:
+Under the hood though, two key differences provide improved performance and usability:
 
 - **lazy materialization**: Unlike traditional data frames, duckplyr defers computation until absolutely necessary, i.e. lazily, allowing DuckDB to optimize execution.
 - **prudence**: Automatic materialization is controllable, as automatic materialization of large data might otherwise inadvertently lead to memory problems.
 
 The term "prudence" is introduced here to set a clear distinction from the concept of "laziness", and because "control of automatic materialization" is a mouthful.
 
-## Eager and lazy computation
+## DuckDB optimization: lazy evaluation
```

Comment:
"lazy evaluation" is easy to confuse with how R handles arguments.

Reply:
oops, this should be "lazy computation", sorry.